cargo-dist/src/errors.rs

```diff
@@ -128,10 +128,22 @@ pub enum DistError {
         #[diagnostic_source]
         inner: AxoprojectError,
     },
+
     /// User tried to enable Github CI support but no url for the repo
     #[error("Github CI support requires you to specify the URL of your repository")]
-    #[diagnostic(help(r#"Set the repository = "https://github.com/..." key in your Cargo.toml"#))]
+    #[diagnostic(help(
+        r#"Set the repository = "https://github.com/..." key in the Cargo.toml of your app"#
```

Comment:
how precise can we get here.... could we know which app they were trying to release and therefore which cargo.toml was missing the repo. given how frequent and blocking this error is i think the extra effort would have a huge impact